extensions/pkg/controller/status_test.go

```diff
@@ -43,20 +44,20 @@ var _ = Describe("Status", func() {
 		lastOpType       = gardencorev1beta1.LastOperationTypeCreate
 		lastOpDesc       = "foo"
 
-		ctrl   *gomock.Controller
-		logger logr.Logger
-		c      *mockclient.MockClient
+		ctrl *gomock.Controller
+		log  logr.Logger
```

Comment:
Just out of curiosity, why did you replace `logger` with `log`?

Reply:
Just for symmetry/harmonization reasons since we used to rename this in all previous similar refactoring PRs as well.